frontend/src/scenes/session-recordings/player/controller/PlayerController.tsx

```diff
@@ -37,68 +38,60 @@ export function PlayerController(): JSX.Element {
-                        tooltip={
-                            <>
-                                {showPause ? 'Pause' : 'Play'}
-                                <KeyboardShortcut space />
-                            </>
-                        }
-                    >
-                        {showPause ? <IconPause className="text-2xl" /> : <IconPlay className="text-2xl" />}
-                    </LemonButton>
-                    <SeekSkip direction="forward" />
-                </div>
-                <div className="flex items-center gap-1 flex-1 justify-end">
-                    <Tooltip title="Playback speed">
-                        <LemonButtonWithDropdown
-                            data-attr="session-recording-speed-select"
-                            dropdown={{
-                                overlay: (
-                                    <div className="space-y-px">
-                                        {PLAYBACK_SPEEDS.map((speedToggle) => (
-                                            <LemonButton
-                                                fullWidth
-                                                active={speed === speedToggle}
-                                                key={speedToggle}
-                                                onClick={() => {
-                                                    setSpeed(speedToggle)
-                                                }}
-                                            >
-                                                {speedToggle}x
-                                            </LemonButton>
-                                        ))}
-                                    </div>
-                                ),
-                                closeOnClickInside: true,
-                            }}
-                            sideIcon={null}
+            <div className="flex justify-between h-8 gap-2 m-2 mt-1">
+                <div className="flex divide-x gap-2">
+                    <Timestamp />
+                    <div className="flex pl-2 gap-1">
+                        <LemonButton
                             size="small"
+                            onClick={togglePlayPause}
+                            tooltip={
+                                <>
+                                    {showPause ? 'Pause' : 'Play'}
+                                    <KeyboardShortcut space />
+                                </>
+                            }
                         >
-                            {speed}x
-                        </LemonButtonWithDropdown>
-                    </Tooltip>
-
-                    <Tooltip title={`Skip inactivity (${skipInactivitySetting ? 'on' : 'off'})`}>
+                            {showPause ? <IconPause className="text-2xl" /> : <IconPlay className="text-2xl" />}
+                        </LemonButton>
+                        <SeekSkip direction="backward" />
```

Comment:
it feels more natural to me to have back, play, forwards

instead of play, back, forwards

but then primary > secondary makes sense too 

so, no strong pushback here

Reply:
I originally had it that way but it looked cleaner to me to have the circular icons next to each other. Adding `data-attrs` here too to understand usage - I wouldn't be opposed to removing one given keyboard shortcuts are supported